number_of_agents: 10
run_duration: 100
flying_height: 1
takeoff_and_land_duration: 3
agent_sensing_radius: 5
control_weights:
  - 0.07
  - 0.01
  - 0.6
  - 0.08
  - 1
  - 1
  - 0.01
agent_max_vel: 1
compute_frequency: 20
cf_radius: 0.20
noise_std: 0.05
publish_rate: 20
